CHANGELOG.md

```diff
@@ -0,0 +1,1730 @@
+# Changelog
+
+<!-- <START NEW CHANGELOG ENTRY> -->
+
+## v1.15.2
```

Comment:
I used [github-activity](https://github.com/executablebooks/github-activity) which is another tool maintained by folks at Jupyter to generate Changelogs. If we use `jupyter-releaser`, this will be generated automatically. Check [here](https://github.com/mahendrapaipuri/jupyter-power-usage/actions/runs/6495732015/job/17641253767#step:3:224)